docfx_project/toc.yml

```diff
@@ -1,5 +1,7 @@
 - name: Home
   href: index.md
+- name: Documentation
```

Comment:
Any reason you chose to add a new folder for docs? This site is meant to be all about docs, so not sure why we need a nested folder for it...

Reply:
Yeah, I was not sure what to call it. So there are 2 sections to the website. The first is the API docs which are auto-generated. The second is all the article-like docs which contains tutorials, concepts, etc .

For the second part I did not want to dump everything in the root of the docs project. 

I am open to other suggestion on what to call those....